dist.html

```diff
@@ -94,8 +94,8 @@ <h3 id="measure-description" style="font-size: 14px; margin: 0 3px"></h3>
                                     <label><input name="cumulative-toggle" type="radio" value="1"> Cumulative</label>
                                 </div>
                                 <div>
-                                    Date range variable: filter submissions by build ID range or submission date range:
-                                    <label><input name="build-time-toggle" type="radio" value="0"> Build Date</label>
+                                    Date range variable: filter submissions by build/release date range or submission date range:
```

Comment:
it really is a build date though. and release date & build date aren't
always the same. plus it's not data only collected from a specific release
day

On Tue, Jul 7, 2015 at 6:02 PM, Anthony Zhang <notifications@github.com>
wrote:

> In dist.html
> <https://github.com/mozilla/telemetry-dashboard/pull/90#discussion_r34096500>
> :
>
> > @@ -94,8 +94,8 @@ <h3 id="measure-description" style="font-size: 14px; margin: 0 3px"></h3>
> >                                      <label><input name="cumulative-toggle" type="radio" value="1"> Cumulative</label>
> >                                  </div>
> >                                  <div>
> > -                                    Date range variable: filter submissions by build ID range or submission date range:
> > -                                    <label><input name="build-time-toggle" type="radio" value="0"> Build Date</label>
> > +                                    Date range variable: filter submissions by build/release date range or submission date range:
>
> For viewing data under the release channel, it might make more sense to
> say "release date" rather than "build date".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/telemetry-dashboard/pull/90/files#r34096500>.
>